entities/backup/descriptor.go

```diff
@@ -255,6 +255,8 @@ type BackupDescriptor struct {
 	CompletedAt   time.Time         `json:"completedAt"`
 	ID            string            `json:"id"` // User created backup id
 	Classes       []ClassDescriptor `json:"classes"`
+	RbacBackups   []byte            `json:"rbacBackups"`
+	UserBackups   []byte            `json:"dynUserBackups"`
```

Comment:
could you rename the json to `UserBackups`

Reply:
removed